scripts/unsupported/combine_tracks_postprocessing_cnv/combine_tracks.wdl

```diff
@@ -0,0 +1,621 @@
+    alleliccapseg_seg_pd['Start.bp'] = model_segments_seg_pd['START']
+    alleliccapseg_seg_pd['End.bp'] = model_segments_seg_pd['END']
+    alleliccapseg_seg_pd['n_probes'] = model_segments_seg_pd['NUM_POINTS_COPY_RATIO_1']
+    alleliccapseg_seg_pd['length'] = alleliccapseg_seg_pd['End.bp'] - alleliccapseg_seg_pd['Start.bp']
+    alleliccapseg_seg_pd['n_hets'] = model_segments_seg_pd['NUM_POINTS_ALLELE_FRACTION']
+
+    #ModelSegments estimates posterior credible intervals, while AllelicCapSeg performs maximum a posteriori (MAP) estimation.
+    #The copy-ratio and allele-fraction models fit by both also differ.
+    #We will attempt a rough translation of the model fits here.
+    #  Next line should be replaced with a function that uses the MS MAF estimates.
+
+    # Old version:
+    # alleliccapseg_seg_pd['f'] = model_segments_seg_pd['MINOR_ALLELE_FRACTION_POSTERIOR_50']
+
+    alleliccapseg_seg_pd['f'] = simple_determine_allelic_fraction(model_segments_seg_pd)
+
+    alleliccapseg_seg_pd['tau'] = 2. * 2**model_segments_seg_pd['LOG2_COPY_RATIO_POSTERIOR_50']
+    alleliccapseg_seg_pd['sigma.tau'] = 2**model_segments_seg_pd['LOG2_COPY_RATIO_POSTERIOR_90'] - 2**model_segments_seg_pd['LOG2_COPY_RATIO_POSTERIOR_10']
+    sigma_f = (model_segments_seg_pd['MINOR_ALLELE_FRACTION_POSTERIOR_90'].values - model_segments_seg_pd['MINOR_ALLELE_FRACTION_POSTERIOR_10'].values) / 2.
+    sigma_mu = np.sqrt(sigma_f**2 + alleliccapseg_seg_pd['sigma.tau']**2) #we propagate errors in the products f * tau and (1 - f) * tau in the usual way
+    alleliccapseg_seg_pd['mu.minor'] = alleliccapseg_seg_pd['f'] * alleliccapseg_seg_pd['tau']
+    alleliccapseg_seg_pd['sigma.minor'] = sigma_mu
+    alleliccapseg_seg_pd['mu.major'] = (1. - alleliccapseg_seg_pd['f']) * alleliccapseg_seg_pd['tau']
+    alleliccapseg_seg_pd['sigma.major'] = sigma_mu
+
+    #For whatever reason, AllelicCapSeg attempts to call CNLOH.  Documentation is spotty, but it seems like it attempts
+    # to distinguish between three states ("0 is flanked on both sides, 1 is one side, 2 is no cn.loh").
+    # Let's just set everything to 2 for now.
+    # Hopefully, ABSOLUTE is robust to this ...
+    alleliccapseg_seg_pd['SegLabelCNLOH'] = 2
+
+    #One important caveat: for segments with less than 10 hets [TODO: Verify], AllelicCapSeg also tries to call whether a segment is "split" or not.
+    #  This script will attempt to call "split" on all segments.
+    # ACS performs a simple hypothesis test on the alternate-allele fractions to see if
+    # a unimodal distribution peaked at 0.5 is supported over a bimodal distribution peaked at f and 1 - f.
+    # If the former is supported, then AllelicCapSeg ignores the MAP estimate of f and simply sets it to be 0.5.
+    # ABSOLUTE may actually be rather sensitive to this.  Again, let's ignore for now, and we can later port this
+    # statistical test if necessary.  I've done it above (ham-fistedly) and also have python code for the stat test
```

Comment:
Looks like these comments were edited.  I'm fine with that, but it's inaccurate to say that the ham-fisted solution above is the same as the port of the statistical test that I've done previously.

Reply:
Done... Just eliminated the last sentence